step1/hackatom-node.sh

```diff
@@ -20,14 +24,20 @@ BUILD_CMD=./cosmos-sdk/build/simd
 VALIDATOR=alice
 CHAIN_ID=hackatom6-chain
 
+# Cleanup previous installations, if any.
 rm -rf $CFG_DIR
+
+# Add ALICE's key to the keyring.
 $BUILD_CMD keys add $VALIDATOR --keyring-backend test
 VALIDATOR_ADDRESS=$($BUILD_CMD keys show $VALIDATOR -a --keyring-backend test)
+
+# Initialize the genesis file. It is available under $CFG_DIR/config/genesis.json.
 $BUILD_CMD init $VALIDATOR --chain-id $CHAIN_ID
-# 30s voting period
-sed -i 's/"voting_period": "172800s"/"voting_period": "30s"/' $CFG_DIR/config/genesis.json
 $BUILD_CMD add-genesis-account $VALIDATOR_ADDRESS 10000000000stake
 $BUILD_CMD gentx $VALIDATOR 1000000000stake --keyring-backend test --chain-id $CHAIN_ID
 $BUILD_CMD collect-gentxs
 
-$BUILD_CMD start
+# Run the node.
+$BUILD_CMD start --minimum-gas-prices 0.0000001stake
```

Comment:
this is a big change I made. without this, fees are 0, which defeats the purpose of feegrant